autoload/ale.vim

```diff
@@ -252,10 +252,10 @@ endfunction
 function! ale#GetLocItemMessage(item, format_string) abort
     let l:msg = a:format_string
     let l:severity = g:ale_echo_msg_warning_str
-    let l:code = get(a:item, 'code', '')
+    let l:code = get(a:item, 'code', -1)
     let l:type = get(a:item, 'type', 'E')
     let l:linter_name = get(a:item, 'linter_name', '')
-    let l:code_repl = !empty(l:code) ? '\=submatch(1) . l:code . submatch(2)' : ''
+    let l:code_repl = l:code != -1 ? '\=submatch(1) . l:code . submatch(2)' : ''
```

Comment:
Undo the changes to this file. The LSP code should be fixed so it doesn't set the key with `-1` instead.